helm/values.yaml

```diff
@@ -412,7 +412,7 @@ nginx:
   image:
     repository: nginxinc/nginx-unprivileged
     pullPolicy: IfNotPresent
-    tag: stable
+    tag: stable@sha256:2fb60041005d5f31df11e7ef6c011ecc71de3a536fd131a650dd18239475861f
```

Comment:
```suggestion
    tag: 1.26@sha256:2fb60041005d5f31df11e7ef6c011ecc71de3a536fd131a650dd18239475861f
```

Reply:
Let's leave this for now, as NGINX is not pinned exactly to that version on the PodiumD platform.